gitlab.go

```diff
@@ -117,6 +117,12 @@ func (t *ISOTime) UnmarshalJSON(data []byte) error {
 	return err
 }
 
+// implement String method for ISOTime
```

Comment:
To satisfy `golint` (and other Go linters) the comment should start with the name of the method. So in this case `String implements the Stringer interface` seems like a good comment...